Code/Framework/AzToolsFramework/AzToolsFramework/AzToolsFrameworkModule.cpp

```diff
@@ -84,6 +87,9 @@ namespace AzToolsFramework
             Prefab::PrefabSystemComponent::CreateDescriptor(),
             Prefab::EditorPrefabComponent::CreateDescriptor(),
             Prefab::ProceduralPrefabSystemComponent::CreateDescriptor(),
+            AzToolsFramework::ViewBookmarkSystemComponent::CreateDescriptor(),
+            AzToolsFramework::SharedViewBookmarkComponent::CreateDescriptor(),
```

Comment:
suggestion: I'd consider not calling `CreateDescriptor` for `SharedViewBookmarkComponent` right now as it's not fully operational, it's probably better we add this back later